### PR TITLE
Saksham/issue 81 test cases: Util-test, loggedInUserData bugfix

### PR DESCRIPTION
**Pull request by sakshambhatt**
## Changes: 
1. index.tsx renamed to Index.tsx
2. added tests for getUserData, updateStatus & updateMarkYourSelfAs_ functions of Util.ts
3. removed unused resources, console logs, and rectified lint errors
4. upgraded jest to 28.1.1
5. bugfix: loggedInUserData remained null even after successful getUserData [detailed description](https://github.com/Real-Dev-Squad/mobile-app/issues/81#issuecomment-1158438716)
6. changed to Util functions to be more readable by using async/await

## Testing:
2. image
5. RdsApp
 
 ## Checklist:
- [x]  I have performed a self-review of my own code
- [x]  My changes generate no new warnings. [Util-test.tsx gives warnings about string and undefined]
- [x]  Any dependent changes have been merged and published in downstream modules

## Reviewers:
@shreya-mishra

### CODE REVIEW

File: __tests__/Util-test.ts [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+describe('updateMarkYourSelfAs_ util', () => {
+  test('pass arg undefined receive throw error', async () => {
+    mockedAxios.patch.mockRejectedValue(
+      // eslint-disable-next-line no-use-before-define
+      `TypeError: Cannot read properties of undefined (reading 'protocol')`,
+    );
+    try {
+      await updateMarkYourSelfAs_(undefined);
+    } catch (err) {
+      /* tslint:disable-next-line */
```
Comment (shreya-mishra) [inline __tests__/Util-test.ts]: remove the eslint disabled comment.

**Review by shreya-mishra — COMMENTED** [review]
- raise different PRs for tests and the other changes.
- rest of all looks good to me.

File: __tests__/Util-test.ts (hunk 1 above)
Comment (sakshambhatt) [reply __tests__/Util-test.ts]: I realized and changed the rule as well as fixed the lint error properly. Thanks for the feedback! :clinking_glasses:

**Review by shreya-mishra — APPROVED** [review]
LGTM

**Pull request merged**